--- debugger_old.py	2017-11-23 08:29:21.937240168 +0000
+++ debugger.py	2017-11-23 08:28:34.478130522 +0000
@@ -1652,7 +1652,7 @@
     cur_bp[(bp.filename, bp.breakpoint_id)] = bp
 
 def try_bind_break_point(mod_filename, module, bp):
-    if module.filename.lower() == path.abspath(bp.filename).lower():
+    if breakpoint_path_match(bp.filename,module.filename):
         bp.filename = mod_filename
         bp.is_bound = True
         add_break_point(bp)
